presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -500,7 +501,7 @@ public void testFailForLongVarcharPartitionColumn()
 
         List<TestColumn> columns = ImmutableList.of(partitionColumn, varcharColumn);
 
-        HiveErrorCode expectedErrorCode = HIVE_INVALID_PARTITION_VALUE;
+        MetastoreErrorCode expectedErrorCode = HIVE_INVALID_PARTITION_VALUE;
```

Comment:
We could do

ErrorCode expectedErrorCode = HIVE_INVALID_PARTITION_VALUE.toErrorCode();

since we have to call `.toErrorCode();` in the checking method anyways